src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -149,6 +149,20 @@ internal partial class Binder
                     unconvertedSource.HasErrors);
             }
 
+            if (conversion.Kind == ConversionKind.InterpolatedStringHandler)
+            {
+                var unconvertedSource = (BoundUnconvertedInterpolatedString)source;
+                return new BoundConversion(
+                    syntax,
+                    BindUnconvertedInterpolatedStringToHandlerType(unconvertedSource, (NamedTypeSymbol)destination, diagnostics, isHandlerConversion: true),
+                    conversion,
+                    @checked: false,
+                    explicitCastInCode: false,
```

Comment:
>explicitCastInCode: false [](http://example.com/codeflow?start=20&length=25)

This looks suspicious.

Reply:
Chris mentioned this earlier. I'm planning on addressing in the next PR, unless you think this needs to be addressed now.

Reply:
Oh, sorry, that was on a different line. Yes, this is incorrect and I'll add a test.

Reply:
> Chris mentioned this earlier. 

This is not about checked/unchecked, which has no impact on this conversion. This is about explicitCastInCode, which is likely to be significant.